war/pom.xml

```diff
@@ -193,23 +193,17 @@ THE SOFTWARE.
           </execution>
         </executions>
       </plugin>
-      <!-- JENKINS-68021: Work around JDK-8206937 by clearing the release=8 flag. -->
```

Comment:
No need for weird compiler options now that we are no longer using deprecated APIs.